extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -333,25 +333,108 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, env => {
       addTestInstance(root3.getElement(body, '#target'), null);
     });
 
-    it('should find an AMP element for AMP search', () => {
+    it('should find an AMP element for AMP search', async () => {
       child.classList.add('i-amphtml-element');
-      return root.getAmpElement(body, '#child').then(element => {
-        expect(element).to.equal(child);
-      });
+      const elements = await root.getAmpElementOrElements(body, '#child');
```

Comment:
I see `getAmpElement` is still used for non experiment case. Please keep the test coverage.
And I don't see `getAmpElementOrElements` declared anywhere. Please verify.

Reply:
Oops, didn't fix the tests because one of the node modules has been acting up. Will fix now.